graphql/mutation_resolver.go

```diff
@@ -193,8 +193,12 @@ func (r *mutationResolver) SaveDistro(ctx context.Context, opts SaveDistroInput)
 		return nil, InputValidationError.Send(ctx, fmt.Sprintf("validating changes for distro '%s': '%s'", d.Id, validationErrs.String()))
 	}
 
-	if err = d.ReplaceOne(ctx); err != nil {
-		return nil, InternalServerError.Send(ctx, err.Error())
+	if err = data.UpdateDistro(ctx, oldDistro, d); err != nil {
```

Comment:
This is not generated and needs review. Also, I changed this to [`UpdateDistro`](https://github.com/evergreen-ci/evergreen/blob/105191bacaf22f9c5d800eb9a4be4d3ffac89730/rest/data/distro.go#L20) because it does a check for the dispatcher version which I thought might be important? But will remove if it's incorrect.

Reply:
Cool catch -- I actually see no reason why UpdateDistro is in rest/data as opposed to model/distro/distro.go; could you try moving it? If that doesn't cause too many problems, it would make it more discoverable.

Reply:
Sadly it gave me an import cycle 😔 But I kind of believe `UpdateDistro` belongs in rest/data because of how it is returning gimlet error responses (which are then consumed by the packages in rest/route), so I feel like it's okay 🤔